packages/grid/test/column-group.test.js

```diff
@@ -79,76 +79,6 @@ describe('column group', () => {
     expect(columns[1].frozen).to.be.true;
   });
 
-  it('should hide group column', () => {
```

Comment:
Grouped the hidden tests into nested suites for `hidden` and `auto-hide`, and also updated the test labels.